src/react/deckgl.js

```diff
@@ -114,7 +128,11 @@ export default class DeckGL extends React.Component {
       cachePosition: false,
       centerOrigin: false,
       onClick: this._onClick,
-      onMouseMove: this._onMouseMove
+      onMouseMove: this._onMouseMove,
+      onDragStart: this._onDragEvent,
+      onDragMove: this._onDragEvent,
+      onDragEnd: this._onDragEvent,
+      onDragCancel: this._onDragCancel
```

Comment:
We definitely should not register events based on `pickable` states at initialization. In most of our use cases the layers are not passed in till data is loaded. Since we're only logging a warning it's not broken at the moment.

I agree this should be fixed in a separate PR.